symengine/__init__.py

```diff
@@ -9,7 +9,10 @@
         UndefFunction, Function, FunctionSymbol as AppliedUndef,
         have_numpy, true, false, Equality, Unequality, GreaterThan,
         LessThan, StrictGreaterThan, StrictLessThan, Eq, Ne, Ge, Le,
-        Gt, Lt)
+        Gt, Lt, GoldenRatio, Catalan, EulerGamma, LambertW,
+        zeta, dirichlet_eta, KroneckerDelta, LeviCivita, erf, erfc,
+        lowergamma, uppergamma, loggamma, beta, polygamma, sign,
+        floor, ceiling, conjugate, digamma, trigamma)
```

Comment:
I forgot that I moved all the functions to `symengine/functions.py`. Can you do that? Sorry about the last comment